libevmasm/AssemblyItem.h

```diff
@@ -162,6 +176,8 @@ class AssemblyItem
 	AssemblyItemType m_type;
 	Instruction m_instruction; ///< Only valid if m_type == Operation
 	std::shared_ptr<u256> m_data; ///< Only valid if m_type != Operation
+	/// If m__type == VerbatimBytecode, this holds stack difference and verbatim bytecode.
+	std::optional<std::pair<int, bytes>> m_verbatimBytecode;
```

Comment:
```suggestion
	std::optional<std::pair<int /*stack difference*/, bytes /*verbatimData*/>> m_verbatimBytecode;
```

Also nitpick: `m__type` -> `m_type`.

Reply:
This is a weird syntax...

Reply:
Just a suggestion for improved documentation. Aren't we using this syntax for some prototypes?

Reply:
I don't really care about this being added or not.

Reply:
Yes, but template type parameters cannot be named. Maybe we should enforce pairs to be structs...